packages/core/test/util/resolvers.test.ts

```diff
@@ -25,7 +25,7 @@
 import { resolveSchema } from '../../src/util/resolvers';
 import test from 'ava';
 
-test('resolveSchema - resolves schema with any ', t => {
+test.only('resolveSchema - resolves schema with any ', t => {
```

Comment:
`only` should be removed from here :smile:

Reply:
oops ;)